pom.xml

```diff
@@ -26,6 +26,7 @@
 		<java.version>1.7</java.version>
 		<spring-cloud-dataflow-ui.version>1.1.0.M2</spring-cloud-dataflow-ui.version>
 		<spring-cloud-deployer.version>1.1.0.BUILD-SNAPSHOT</spring-cloud-deployer.version>
```

Comment:
I think we loosely follow git repo name for these versions. `spring-cloud-deployer.version` is used elsewhere so we prob should not change it.